src/domain/Lotto/rule.js

```diff
@@ -1,22 +1,22 @@
-import { RULES } from "../../util/rule.js";
-
-const LOTTO_RULES = {
-  purChasePriceRule: (number) =>
-    number >= RULES.MIN_LOTTO_PURCHASE_PRICE &&
-    number <= RULES.MAX_LOTTO_PURCHASE_PRICE,
-  winningNumberRule: (numbers) =>
-    Array.isArray(numbers) &&
-    new Set(numbers).size === RULES.TICKET_LENGTH &&
-    numbers.every(
-      (number) =>
-        Number.isInteger(number) &&
-        number >= RULES.MIN_TICKET_NUMBER &&
-        number <= RULES.MAX_TICKET_NUMBER,
-    ),
-  bonusNumberRule: (val) =>
-    Number.isInteger(val) &&
-    val >= RULES.MIN_TICKET_NUMBER &&
-    val <= RULES.MAX_TICKET_NUMBER,
-};
-
-export default LOTTO_RULES;
+import RULES from "../../util/rule.js";
+
+const LOTTO_RULES = {
+  purChasePriceRule: (number) =>
+    number >= RULES.MIN_LOTTO_PURCHASE_PRICE &&
+    number <= RULES.MAX_LOTTO_PURCHASE_PRICE,
+  winningNumberRule: (numbers) =>
+    Array.isArray(numbers) &&
+    new Set(numbers).size === RULES.TICKET_LENGTH &&
+    numbers.every(
+      (number) =>
+        Number.isInteger(number) &&
+        number >= RULES.MIN_TICKET_NUMBER &&
+        number <= RULES.MAX_TICKET_NUMBER,
+    ),
+  bonusNumberRule: (val) =>
+    Number.isInteger(val) &&
+    val >= RULES.MIN_TICKET_NUMBER &&
+    val <= RULES.MAX_TICKET_NUMBER,
+};

```

Comment:
소소한 부분인데 개행이 하나씩 들어가주면 읽기 더 좋을거 같네용

```suggestion
const LOTTO_RULES = {
  purChasePriceRule: (number) =>
    number >= RULES.MIN_LOTTO_PURCHASE_PRICE &&
    number <= RULES.MAX_LOTTO_PURCHASE_PRICE,
    
  winningNumberRule: (numbers) =>
    Array.isArray(numbers) &&
    new Set(numbers).size === RULES.TICKET_LENGTH &&
    numbers.every(
      (number) =>
        Number.isInteger(number) &&
        number >= RULES.MIN_TICKET_NUMBER &&
        number <= RULES.MAX_TICKET_NUMBER,
    ),
    
  bonusNumberRule: (val) =>
    Number.isInteger(val) &&
    val >= RULES.MIN_TICKET_NUMBER &&
    val <= RULES.MAX_TICKET_NUMBER,
};

```